go-controller/pkg/factory/factory.go

```diff
@@ -1195,6 +1267,24 @@ func (wf *WatchFactory) GetEgressFirewall(namespace, name string) (*egressfirewa
 	return egressFirewallLister.EgressFirewalls(namespace).Get(name)
 }
 
+// GetEgressFirewalls returns a list of all the egress firewalls in the cluster
+func (wf *WatchFactory) GetEgressFirewalls() ([]*egressfirewallapi.EgressFirewall, error) {
+	egressFirewallLister := wf.informers[EgressFirewallType].lister.(egressfirewalllister.EgressFirewallLister)
+	return egressFirewallLister.List(labels.Everything())
+}
+
+// GetDNSNameResolver gets a specific dns name resolver by the namespace/name
+func (wf *WatchFactory) GetDNSNameResolver(namespace, name string) (*ocpnetworkapiv1alpha1.DNSNameResolver, error) {
+	dnsNameResolverLister := wf.informers[DNSNameResolverType].lister.(ocpnetworklister.DNSNameResolverLister)
+	return dnsNameResolverLister.DNSNameResolvers(namespace).Get(name)
+}
+
+// GetDNSNameResolvers returns a list of all the dns name resolvers in the namespace
+func (wf *WatchFactory) GetDNSNameResolvers(namespace string) ([]*ocpnetworkapiv1alpha1.DNSNameResolver, error) {
+	dnsNameResolverLister := wf.informers[DNSNameResolverType].lister.(ocpnetworklister.DNSNameResolverLister)
+	return dnsNameResolverLister.DNSNameResolvers(namespace).List(labels.Everything())
+}
+
```

Comment:
I think these functions can be replaced with local lister calls (this is what we prefer to do for the new controllers)

Reply:
Removed the functions and replaced them with local lister calls.